src/implementations/twig/components/description-list/description-list.html.twig

```diff
@@ -67,7 +80,12 @@
 
           {# taxonomy #}
           {% if _item.type is not empty and _item.type == 'taxonomy' %}
-            <dd class="ecl-description-list__definition ecl-description-list__definition--taxonomy">
+            <dd
+              class="ecl-description-list__definition ecl-description-list__definition--taxonomy"
+            {%- if _visible_items > 0 -%}
+              data-ecl-description-list-collapsible
```

Comment:
does it count as markup change?

Reply:
not really, to be honest, we call "markup change" something that needs to be done at the markup level, here eUI will entirely rebuild this functionality, not using our attributes since they don't use our js..
We can still add the label to raise attention on a functionality that they will have to "implement", though